free5gc-operator/controllers/free5gc_smfuerouting.go

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+var Uerouting string = `
+info:
+  version: 1.0.1
+  description: Routing information for UE
+ueRoutingInfo:
+  UE1:
+    members:
+    - imsi-208930000000003
```

Comment:
I don't know what all these things mean, but they look like things we should be constructing based on the SMFDeployment.